src/test/java/org/jabref/logic/importer/fetcher/TitleFetcherTest.java

```diff
@@ -44,7 +44,7 @@ public void testGetName() {
 
     @Test
```

Comment:
it would be OK for me to remove these kind of tests completely. Think, the have been introduced in 2015 by students to increase test coverage.